opensrp-chw/src/main/java/org/smartregister/chw/repository/HomeVisitRepository.java

```diff
@@ -125,9 +125,9 @@ public void add(HomeVisit homeVisit) {
             if (StringUtils.isBlank(homeVisit.getSyncStatus())) {
                 homeVisit.setSyncStatus(TYPE_Unsynced);
             }
-            if (StringUtils.isBlank(homeVisit.getFormSubmissionId())) {
-                homeVisit.setFormSubmissionId(generateRandomUUIDString());
-            }
+//            if (StringUtils.isBlank(homeVisit.getFormSubmissionId())) {
```

Comment:
Will this be needed sometime later or in the future?

Reply:
will be delete. let me share the issue status is it solve the duplicate issue?

Reply:
Yes, I've seen the duplicate issue is solved.

Reply:
cool i removed the commented code. 
please approved the changes then i can merge it in master.